frontend/src/components/CardEvent.jsx

```diff
@@ -0,0 +1,34 @@
+const CardEvent = ({data}) => {
+    console.log(data.image)
+    return(
+      <div className="max-w-sm rounded overflow-hidden shadow-lg bg-white h-[20rem] w-[15rem] hover:bg-neutral-300">
+        <img className="h-[10rem] w-[15rem]" src={data.image} onerror="this.src='https://www.shutterstock.com/image-vector/default-ui-image-placeholder-wireframes-600nw-1037719192.jpg';" />
+        <div className="px-3 py-4 flex flex-col justify-center items-center">
+          <div className="font-bold text-base mb-2 mx-[1.3rem]">{data.name}</div>
+          <p className="text-gray-700 text-sm mx-[1.1rem]">
+            📍{data.location}
+          </p>
+
+
```

Comment:
Empty line. Did you run prettifier on your new files?

Reply:
Done